backend/migrations/2023-04-12-162600_rename_plants_columns/down.sql

```diff
@@ -0,0 +1,21 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Please squash all plants SQL files as done in https://github.com/ElektraInitiative/PermaplanT/pull/232 (with check if they are identical to these files here).

Otherwise it is difficult for me to check these SQL files.

Reply:
Ok, I prepared them and moved to another folder